queue_job_chunk/models/queue_job_chunk.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright (c) Akretion 2020
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+from odoo import api, fields, models
+
+from odoo.addons.queue_job.job import job
+
+
+class QueueJobChunk(models.Model):
+    _name = "queue.job.chunk"
+    _description = "Queue Job Chunk"
+
+    def _get_default_company_id(self):
```

Comment:
TODO add a test that
- create 2 queue_job_chunk (with a multi create on a record that have the field company like res.partner)
- check the company
- check the reference
- check that a job have been created

same case but with an object that do not have company like ir.module